src/app/lib/config/services/amharic.js

```diff
@@ -45,29 +45,25 @@ const service = {
     error: {
       404: {
         statusCode: '404',
-        title: '404 - ገጹ አልተገኘም',
-        message:
-          'ይህ የድረገጽ አድራሻውን በትክክል ስላላስገቡ ሊሆን ይችላል፡፡ እባክዎትን አድራሻውን እና ፊደላቱን ያረጋግጡ፡፡',
+        title: '404 - ገፁን አሁን ማግኘት አይችሉም',
```

Comment:
I don't think the `404 - ` is required here because the design of the page already has a static 404 heading. Also it's not in the translation sheet like that.

Reply:
Thanks Jonathan